index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Please give a good title for your project

Reply:
Please, always includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Please give a good title for your project*
* This request lacks a "why": *Please, ensure that you always give a good title with the title tag as this will help search engines perform better at finding your page*